### PR TITLE
[Github Action] Setup phpstan

### LINKED ISSUE #4: [Github Action] Setup phpstan

### PR DESCRIPTION
**Pull request by kingjia90**
Resolves https://github.com/pimcore/web-to-print-bundle/issues/4

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -34,6 +34,7 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode = $treeBuilder->getRootNode();
         $rootNode->addDefaultsIfNotSet();
 
+        // @phpstan-ignore-next-line
```
Comment (blankse) [inline src/DependencyInjection/Configuration.php]: @kingjia90 We should add the symfony phpstan module like in pimcore/pimcore

Comment (kingjia90) [reply src/DependencyInjection/Configuration.php]: i've force pushed the commit but can revert this change to show what happens. 
I've checked some other bundle (headless document) adding this ignore-next-line , maybe it should be something in the neon file instead, right?

I didn't get the symfony module issue point, were you referring to this?
https://github.com/pimcore/google-marketing-bundle/pull/16/files?w=1#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R23

Comment (kingjia90) [reply src/DependencyInjection/Configuration.php]: found a public bundle that uses the same line
https://github.com/pimcore/advanced-object-search/blob/2f62a7148323e5017e9a6b7a20109a2d5d3a4286/src/DependencyInjection/Configuration.php#L36

Comment (blankse) [reply src/DependencyInjection/Configuration.php]: You need this lines:
https://github.com/pimcore/pimcore/blob/fa3c81e6d0b688b5d1f0aaeb04f9e752562c5cd9/composer.json#L142
https://github.com/pimcore/pimcore/blob/fa3c81e6d0b688b5d1f0aaeb04f9e752562c5cd9/phpstan.neon#L2

Comment (kingjia90) [reply src/DependencyInjection/Configuration.php]: ahh cool, didn't know that fixes these. 
Thank you!

File: .github/workflows/static-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+on:
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
```
Comment (robertSt7) [inline .github/workflows/static-analysis.yml:14]:
```suggestion
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'public/**'
```

File: .github/workflows/static-analysis.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+name: "Static Analysis"
+
+on:
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
```
Comment (robertSt7) [inline .github/workflows/static-analysis.yml:7]:
```suggestion
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'public/**'
```

**Review by robertSt7 — APPROVED** [review]

**Pull request merged**